Library/Homebrew/manpages.rb

```diff
@@ -115,14 +121,14 @@ def self.cmd_parser_manpage_lines(cmd_parser)
       lines
     end
 
+    sig { params(cmd_path: Pathname).returns(T.nilable(T::Array[String])) }
     def self.cmd_comment_manpage_lines(cmd_path)
       comment_lines = cmd_path.read.lines.grep(/^#:/)
       return if comment_lines.empty?
-      return if comment_lines.first.include?("@hide_from_man_page")
+      return if T.must(comment_lines.first).include?("@hide_from_man_page")
 
-      lines = [format_usage_banner(comment_lines.first).chomp]
-      comment_lines.slice(1..-1)
-                   .each do |line|
+      lines = [format_usage_banner(T.must(comment_lines.first)).chomp]
```

Comment:
```suggestion
      lines = [format_usage_banner(first_comment_line).chomp]
```
Nice to avoid `T.must` when possible, particularly when it's needed multiple times on the same variable.